sys-account/Makefile

```diff
@@ -27,5 +27,12 @@ this-print: ## print
 	#$(MAKE) srv-print
 
 ## This builds everything
-this-build: ##
-	$(MAKE) flu-gen-lang
+this-build: #go-gen-dep
+	@echo "Generating GRPC models and services for Go & Flutter"
+	@mkdir -p server/rpc/v2
+	@mkdir -p client/lib/api/v2
+	@go generate
+	#$(MAKE) flu-gen-lang
+
+## Build and Run Server Example (Temporary until structure/stack is fixed)
```

Comment:
temporary `this-run-example` recipe (will be removed when structure /stack is fixed).